paddle/fluid/operators/listen_and_serv_op.cc

```diff
@@ -29,6 +29,28 @@ void RunServer(std::shared_ptr<detail::AsyncGRPCServer> service) {
   VLOG(4) << "RunServer thread end";
 }
 
+namespace {
+
+bool global_exit_flag = false;
+
+typedef std::unordered_set<std::shared_ptr<detail::ReceivedQueue>>
+    BlockingQueueSet;
+BlockingQueueSet global_blocking_queue_set;
+
+// use unnamed namespace to avoid accessing from outside this file
+void StopAndExit(int signal_num) {
```

Comment:
Can we bind this signal handler into the RPC server or somewhere so that we don't need to add globals?

Reply:
I will put the StopAndExit Function and globals into a Class, (Actually is the same with unnamed namespace)